### ISSUE TITLE
Homebrew build error

### ISSUE THREAD

**Issue by mogol** [tarantool/tarantool#1977]
Couldn't build via Homebrew on OS X El Capitan/Sierra , Xcode8.1

```
➜  brew install tarantool --HEAD
Updating Homebrew...
==> Using the sandbox
==> Cloning https://github.com/tarantool/tarantool.git
Updating /Users/mogol/Library/Caches/Homebrew/tarantool--git
==> Checking out branch 1.7
Synchronizing submodule url for 'lib/msgpack-python'
Synchronizing submodule url for 'lib/tarantool-python'
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/tarantool/HEAD-aeec869_2 
==> make
Last 15 lines from /Users/mogol/Library/Logs/Homebrew/tarantool/02.make:
[ 85%] Building C object src/CMakeFiles/server.dir/__/third_party/lua-cjson/lua_cjson.c.o
cd /tmp/tarantool-20161207-42039-1olb47a/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang  -DCORO_ASM -DNDEBUG -DNVALGRIND -D__STDC_CONSTANT_MACROS=1 -D__STDC_FORMAT_MACROS=1 -D__STDC_LIMIT_MACROS=1 -I/tmp/tarantool-20161207-42039-1olb47a/src -I/tmp/tarantool-20161207-42039-1olb47a/src/lib -I/tmp/tarantool-20161207-42039-1olb47a/src/lib/small -I/tmp/tarantool-20161207-42039-1olb47a -I/tmp/tarantool-20161207-42039-1olb47a/third_party/zstd/lib -I/tmp/tarantool-20161207-42039-1olb47a/third_party/zstd/lib/common -I/usr/local/opt/openssl/include -I/tmp/tarantool-20161207-42039-1olb47a/third_party -I/tmp/tarantool-20161207-42039-1olb47a/third_party/coro -I/tmp/tarantool-20161207-42039-1olb47a/third_party/gopt -I/tmp/tarantool-20161207-42039-1olb47a/third_party/luajit/src -I/usr/local/opt/readline/include -I/tmp/tarantool-20161207-42039-1olb47a/third_party/libyaml/include -I/tmp/tarantool-20161207-42039-1olb47a/src/lib/msgpuck -I/tmp/tarantool-20161207-42039-1olb47a/third_party/libutil_freebsd  -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-unused-value -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.11   -o CMakeFiles/server.dir/__/third_party/lua-cjson/lua_cjson.c.o   -c /tmp/tarantool-20161207-42039-1olb47a/third_party/lua-cjson/lua_cjson.c
[ 85%] Building C object src/CMakeFiles/server.dir/__/third_party/lua-cjson/strbuf.c.o
cd /tmp/tarantool-20161207-42039-1olb47a/src && /usr/local/Homebrew/Library/Homebrew/shims/super/clang  -DCORO_ASM -DNDEBUG -DNVALGRIND -D__STDC_CONSTANT_MACROS=1 -D__STDC_FORMAT_MACROS=1 -D__STDC_LIMIT_MACROS=1 -I/tmp/tarantool-20161207-42039-1olb47a/src -I/tmp/tarantool-20161207-42039-1olb47a/src/lib -I/tmp/tarantool-20161207-42039-1olb47a/src/lib/small -I/tmp/tarantool-20161207-42039-1olb47a -I/tmp/tarantool-20161207-42039-1olb47a/third_party/zstd/lib -I/tmp/tarantool-20161207-42039-1olb47a/third_party/zstd/lib/common -I/usr/local/opt/openssl/include -I/tmp/tarantool-20161207-42039-1olb47a/third_party -I/tmp/tarantool-20161207-42039-1olb47a/third_party/coro -I/tmp/tarantool-20161207-42039-1olb47a/third_party/gopt -I/tmp/tarantool-20161207-42039-1olb47a/third_party/luajit/src -I/usr/local/opt/readline/include -I/tmp/tarantool-20161207-42039-1olb47a/third_party/libyaml/include -I/tmp/tarantool-20161207-42039-1olb47a/src/lib/msgpuck -I/tmp/tarantool-20161207-42039-1olb47a/third_party/libutil_freebsd  -fexceptions -funwind-tables -fno-common -fopenmp -msse2 -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-unused-value -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.11   -o CMakeFiles/server.dir/__/third_party/lua-cjson/strbuf.c.o   -c /tmp/tarantool-20161207-42039-1olb47a/third_party/lua-cjson/strbuf.c
[ 86%] Linking CXX static library libserver.a
cd /tmp/tarantool-20161207-42039-1olb47a/src && /usr/local/Cellar/cmake/3.7.1/bin/cmake -P CMakeFiles/server.dir/cmake_clean_target.cmake
cd /tmp/tarantool-20161207-42039-1olb47a/src && /usr/local/Cellar/cmake/3.7.1/bin/cmake -E cmake_link_script CMakeFiles/server.dir/link.txt --verbose=1
/usr/bin/ar qc libserver.a  CMakeFiles/server.dir/find_path.c.o CMakeFiles/server.dir/sio.cc.o CMakeFiles/server.dir/evio.cc.o CMakeFiles/server.dir/coio.cc.o CMakeFiles/server.dir/coeio.c.o CMakeFiles/server.dir/iobuf.cc.o CMakeFiles/server.dir/coio_buf.cc.o CMakeFiles/server.dir/pickle.c.o CMakeFiles/server.dir/ipc.c.o CMakeFiles/server.dir/latch.c.o CMakeFiles/server.dir/errinj.c.o CMakeFiles/server.dir/fio.c.o CMakeFiles/server.dir/crc32.c.o CMakeFiles/server.dir/random.c.o CMakeFiles/server.dir/scramble.c.o CMakeFiles/server.dir/opts.c.o CMakeFiles/server.dir/cfg.c.o CMakeFiles/server.dir/cpu_feature.c.o CMakeFiles/server.dir/tt_uuid.c.o CMakeFiles/server.dir/uri.c.o CMakeFiles/server.dir/backtrace.cc.o CMakeFiles/server.dir/proc_title.c.o CMakeFiles/server.dir/coeio_file.c.o CMakeFiles/server.dir/clock.c.o CMakeFiles/server.dir/lua/console.c.o CMakeFiles/server.dir/lua/digest.c.o CMakeFiles/server.dir/lua/init.c.o CMakeFiles/server.dir/lua/fiber.c.o CMakeFiles/server.dir/lua/trigger.c.o CMakeFiles/server.dir/lua/ipc.c.o CMakeFiles/server.dir/lua/msgpack.c.o CMakeFiles/server.dir/lua/utils.c.o CMakeFiles/server.dir/lua/errno.c.o CMakeFiles/server.dir/lua/socket.c.o CMakeFiles/server.dir/lua/pickle.c.o CMakeFiles/server.dir/lua/fio.c.o CMakeFiles/server.dir/lua/crypto.c.o CMakeFiles/server.dir/lua/init.lua.c.o CMakeFiles/server.dir/lua/fiber.lua.c.o CMakeFiles/server.dir/lua/buffer.lua.c.o CMakeFiles/server.dir/lua/uuid.lua.c.o CMakeFiles/server.dir/lua/crypto.lua.c.o CMakeFiles/server.dir/lua/digest.lua.c.o CMakeFiles/server.dir/lua/msgpackffi.lua.c.o CMakeFiles/server.dir/lua/uri.lua.c.o CMakeFiles/server.dir/lua/console.lua.c.o CMakeFiles/server.dir/lua/socket.lua.c.o CMakeFiles/server.dir/lua/errno.lua.c.o CMakeFiles/server.dir/lua/log.lua.c.o CMakeFiles/server.dir/lua/help.lua.c.o CMakeFiles/server.dir/lua/help_en_US.lua.c.o CMakeFiles/server.dir/lua/tap.lua.c.o CMakeFiles/server.dir/lua/fio.lua.c.o CMakeFiles/server.dir/lua/csv.lua.c.o CMakeFiles/server.dir/lua/strict.lua.c.o CMakeFiles/server.dir/lua/clock.lua.c.o CMakeFiles/server.dir/lua/title.lua.c.o CMakeFiles/server.dir/lua/argparse.lua.c.o CMakeFiles/server.dir/lua/env.lua.c.o CMakeFiles/server.dir/__/third_party/luafun/fun.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/bc.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/bcsave.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/dis_x86.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/dis_x64.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/dump.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/vmdef.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/v.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/p.lua.c.o CMakeFiles/server.dir/__/third_party/luajit/src/jit/zone.lua.c.o CMakeFiles/server.dir/__/third_party/lua-yaml/lyaml.cc.o CMakeFiles/server.dir/__/third_party/lua-yaml/b64.c.o CMakeFiles/server.dir/__/third_party/lua-cjson/lua_cjson.c.o CMakeFiles/server.dir/__/third_party/lua-cjson/strbuf.c.o
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: libserver.a(coio_buf.cc.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: libserver.a(pickle.c.o) has no symbols
/usr/bin/ranlib libserver.a
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: libserver.a(coio_buf.cc.o) has no symbols
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: libserver.a(pickle.c.o) has no symbols
[ 86%] Built target server
make: *** [all] Error 2
```

**Comment by rtsisyk** [comment]
I can't find the initial error message in this output. Please post actual error message.

**Comment by bigbes** [comment]
I've pasted it into chat

**Comment by rtsisyk** [comment]
okay, I've fixed this problem by  d91264f9e29283a814f51649a9ce10196cc19e81
